pyfr/solvers/base/system.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, backend, rallocs, mesh, initsoln, nregs, cfg):
         # I/O banks for the elements
         self.eles_scal_upts_inb = eles.scal_upts_inb
         self.eles_scal_upts_outb = eles.scal_upts_outb
-        self.eles_vect_upts = eles._vect_upts
+        self.eles_vect_upts = getattr(eles, '_vect_upts', None)
```

Comment:
Not sure I understand this change

Reply:
We don't have a _vect_upts storage in Euler solver when flux AA is in use, and so this line raises an error when user attempts to run an Euler case with flux AA on.

Reply:
I'd put this in an if statement so we only assign if it exists.